Cargo.toml

```diff
@@ -163,6 +163,7 @@ nix = "=0.24.2"
 fwdansi = "=1.1.0"
 winres = "=0.1.12"
 winapi = "=0.3.9"
+winmm-sys = "0.2.0"
```

Comment:
I think we can replace this with the `windows-sys` crate? It's a microsoft crate already found in our dependencies tree.

Reply:
Good idea, done.